Apps/stirling-pdf/docker-compose.yml

```
@@ -0,0 +1,86 @@
+
+    # Image to be used for the container specifies the stirling-pdf version and source
+    image: stirlingtools/stirling-pdf:0.35.0
+
+    # Container restart policy - restarts the container unless manually stopped
+    restart: unless-stopped
+
+    # Environment variables for service configuration
+    environment:
+      # Set to true to download security jar (required for authentication login)
+      - DOCKER_ENABLE_SECURITY=false
+      
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)